en/02_Developer_Guides/08_Performance/index.md

```diff
@@ -7,4 +7,21 @@ icon: tachometer-alt
 
 # Performance
 
+Making your Silverstripe CMS applications fast and scalable is crucial for providing a great user experience and reducing server costs. This section provides guidance on various performance optimization techniques, ranging from caching strategies to database optimizations.
+
+## Key performance considerations
+
+- Implement caching strategies to reduce database queries and server load. See [Partial template caching](partial_template_caching), [Object caching](object_caching), and [HTTP cache headers](http-cache-headers).
+- Optimize your database queries and schema to improve ORM performance. See [ORM performance](orm).
+- Use eager loading to avoid the N+1 query problem and improve database query efficiency. See [Eager loading](eager_loading).
+- Use read-only database replicas to offload read queries from the primary database. See [Read-only database replicas](read-only-database-replicas).
+- Explore other backend performance optimizations, such as increasing server resources, enabling opcode caching, using a CDN, and using a reverse proxy. See [Other optimizations](other_optimisations).
+
+## Where to start
+
+- Use profiling tools to identify the slowest parts of your application. A good starting point it to simply view the page response time in the network panel in your browsers web developer tools.
+- Implementing caching strategies for frequently accessed data and static assets.
+- Focus on optimizing the slowest database queries and adding appropriate indexes.
```

Comment:
```suggestion
- Focus on quick-wins first such as adding appropriate indexes.
```
Not necessarily just slow db queries - but anything that is a quick win should be the first focus. The quick wins for a given project may have nothing to do with the database, in some cases.